.gitignore

```diff
@@ -1,3 +1,4 @@
+/**/gh.exe
```

Comment:
If you compile `gh.exe` under `bin/`, as per our Makefile, doesn't it get ignored as with everything else under `bin/`?

Reply:
ah true, I actually forgot about script/build.go and kept `cd`ing and running `go build`.